gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -1240,6 +1312,12 @@ GbGradeTable.renderTable = function (elementId, tableData) {
   .on("click", ".gb-dropdown-menu .edit-assignment-details", function () {
     const cellElement = $(this).closest(".tabulator-cell, .tabulator-col");
 
+    // Persist current horizontal position so we can restore after the page reloads from the edit
+    const tableContainer = document.querySelector(".tabulator-tableholder");
+    if (tableContainer) {
+      sessionStorage.setItem(GB_SCROLL_LEFT_KEY, tableContainer.scrollLeft);
+    }
+
```

Comment:
we already have focusColumnForAssignmentId() for adding the column. shouldn't we use the same for editing as well?